### PR TITLE
Get ready to release ubuntu-server-netboot 0.1.1

### PR DESCRIPTION
**Pull request by tai271828**
## Types of changes
- **Documentation Update**

## Description
docs(package): rename usn and update meta data

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
- [x] Update the documentation if necessary

## Steps to Test This Pull Request
1. `poetry build`
2. one-off: `poetry config repositories.testpypi https://test.pypi.org/legacy/`
3. `poetry publish -r testpypi` to release to testing/staging pypi https://test.pypi.org/project/ubuntu-server-netboot/
4. `poetry publish` to release to official/production https://pypi.org/project/ubuntu-server-netboot/

## Expected behavior
page on testpypi https://test.pypi.org/project/ubuntu-server-netboot/0.1.1rc3/
page on pypi https://pypi.org/project/ubuntu-server-netboot/0.1.1rc3/

Some fields highlighted to be noticed when reviewing:
- package name should be `ubuntu-server-netboot` (not `usn`)
- licence
- fields of clissifiers
- rendering of package description (should be the same as `README`)

## Additional Information
- If everything looks good, then `pip install ubuntu-server-boot` could make `ubuntu-server-boot` command line tool ready in your system without worrying manual dependency setup.
- When this pull request is landed, the version will bump from `0.1.1rc3` to `0.1.1`.

## Related Issue

issue: #18

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+packages = [
+    {include = "usn"}
+]
+version = "0.1.1rc3"
 description = "This utility generates a netboot directory tree from an Ubuntu Server Live ISO image, an image based on the subiquity installer."
 authors = ["dann frazier <dannf@ubuntu.com>", "Taihsiang Ho (tai271828) <taihsiangho@ubuntu.com>"]
+license = "GPL-2.0-only"
+readme = "README.md"
+classifiers = [
+    "Topic :: Internet :: File Transfer Protocol (FTP)",
```
Comment (dannf) [inline pyproject.toml]: I'm not sure FTP is really a good fit - what about "Topic :: System :: Installation/Setup" maybe?

Comment (tai271828) [reply pyproject.toml]: `Installation/Setup` is way much better. I did not notice this option. Thanks for heading up! The pull request is updated and published as 0.1.1rc4. See this page to check what it looks like https://test.pypi.org/project/ubuntu-server-netboot/0.1.1rc4/

**Comment by tai271828** [comment]
The pull request is ready to review again. If everything goes well, I will merge the code (if I have merge permission then) and publish to PyPI with the version `0.1.1`. See the PyPI rc page https://pypi.org/project/ubuntu-server-netboot/0.1.1rc4/

**Review by dannf — APPROVED** [review]
Thanks @tai271828 ! Feel free to merge (I think you should have an invite to do so)

**Pull request merged**